src/Service.Host/client/src/components/reports/ImpbookIprReport.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+    Loading,
+    ReportTable,
+    BackButton,
+    ExportButton
+} from '@linn-it/linn-form-components-library';
+import Grid from '@material-ui/core/Grid';
+import PropTypes from 'prop-types';
+import Page from '../../containers/Page';
+
+const handleBackClick = (history, options) => {
+    const uri = `/logistics/import-books/ipr?fromDate=${encodeURIComponent(
+        options.fromDate
+    )}&toDate=${encodeURIComponent(options.toDate)}`;
+
+    history.push(uri);
+};
+
+const AssemblyFailsDetails = ({ reportData, loading, history, options, config }) => (
```

Comment:
lots of references to assemblyfails details might want to find/replace them to impbookipr